config/dev.exs

```diff
@@ -0,0 +1,17 @@
+use Mix.Config
+
+config :dogma,
+
+  # Select a set of rules as a base
+  rule_set: Dogma.RuleSet.All,
+
+  # Pick paths not to lint
+  exclude: [
+    ~r(\Alib/vendor/),
+  ],
+
+  # Override an existing rule configuration
+  override: %{
+    FunctionArity => [ max: 6 ],
+    LineLength    => [ max_length: 85 ],
```

Comment:
Feel free to just arbitrarily set that to 100, if that's your preference. I just did as conservative as I could while still making things pass.